src/components/Person/Person.jsx

```diff
@@ -1 +1,43 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const marriageStatus = (
+    sex === 'm'
+      ? 'wife'
+      : 'husband'
+  );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age
+        && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+        )
+      }
+      <p className="Person__partner">
+        {
+          isMarried
+            ? `${partnerName} is my ${marriageStatus}`
+            : 'I am not married'
+        }
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  person: '',
+};
```

Comment:
1. No need to specify default props here;
2. If you want to add default props, type of person is not a string.